WordPress/Sniffs/WP/DeprecatedParameterValuesSniff.php

```diff
@@ -0,0 +1,231 @@
+
+	/**
+	 * Array of function, argument, and replacement function for deprecated argument.
+	 *
+	 * The list of deprecated parameter values can be found by
+	 * looking for `_deprecated_argument()`, `apply_filters_deprecated()`
+	 * and `do_action_deprecated()`.
+	 * The list is sorted first by the WordPress version when deprecated
+	 * and then alphabetically.
+	 * Last updated for WordPress 4.9.6.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var array Multidimensional array with parameter details.
+	 *    $target_functions = array(
+	 *        (string) Function name. => array(
+	 *            (int) Target parameter position, 1-based. => array(
+	 *                (string) Parameter value. => array(
+	 *                'alt'     => (string) Suggested alternative.
+	 *                'version' => (int) The WordPress version when deprecated.
+	 *            )
+	 *         )
+	 *    );
+	 */
+	protected $target_functions = array(
+		'bloginfo' => array(
+			1 => array(
+				'home' => array(
+					'alt'     => 'echo esc_url( home_url() )',
+					'version' => '2.2.0',
+				),
+				'siteurl' => array(
+					'alt'     => 'echo esc_url( home_url() )',
+					'version' => '2.2.0',
+				),
+				'text_direction' => array(
+					'alt'     => 'is_rtl()',
+					'version' => '2.2.0',
+				),
+			),
+		),
+		'get_bloginfo' => array(
+			1 => array(
+				'home' => array(
+					'alt'     => 'home_url()',
+					'version' => '2.2.0',
+				),
+				'siteurl' => array(
+					'alt'     => 'home_url()',
+					'version' => '2.2.0',
+				),
+				'text_direction' => array(
+					'alt'     => 'is_rtl()',
+					'version' => '2.2.0',
+				),
+			),
+		),
+		'add_settings_field' => array(
+			4 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'add_settings_section' => array(
+			4 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'register_setting' => array(
+			1 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'unregister_setting' => array(
+			1 => array(
+				'misc' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.0.0',
+				),
+				'privacy' => array(
+					'alt'     => 'another settings group',
+					'version' => '3.5.0',
+				),
+			),
+		),
+		'do_action_deprecated' => array(
+			1 => array(
+				'refresh_blog_details' => array(
+					'alt'     => 'the action "clean_site_cache"',
+					'version' => '4.9.0',
+				),
+			),
+		),
+		'do_action_deprecated' => array(
```

Comment:
This entry needs to be merged with the previous one. Also see my remark about the sort order.